docs/api-docs/storefront/graphql/graphql-site-content.mdx

```diff
@@ -0,0 +1,1004 @@
+                "entityId": 2,
+                "name": "Shipping & Returns",
+                "parentEntityId": null,
+                "seo": {
+                  "pageTitle": "Shipping & Returns",
+                  "metaKeywords": "Shipping and Return Policy",
+                  "metaDescription": "Shipping and Return Policy"
+                },
+                "plainTextSummary": "To edit this page, log in to your control panel and go to Storefront › Web Pages. Click Edit next...",
+                "path": "/shipping-returns/"
+              }
+            },
+            {
+              "node": {
+                "__typename": "BlogIndexPage",
+                "entityId": 3,
+                "name": "Blogs for the store",
+                "parentEntityId": null,
+                "seo": {
+                  "pageTitle": "Blogs for the store",
+                  "metaKeywords": "",
+                  "metaDescription": "Blogs about top products"
+                },
+                "path": "/blogs/"
+              }
+            },
+            {
+              "node": {
+                "__typename": "ContactPage",
+                "entityId": 6,
+                "name": "Contact Us Form",
+                "parentEntityId": null,
+                "seo": {
+                  "pageTitle": "Contact Us Now!!",
+                  "metaKeywords": "Contact Us",
+                  "metaDescription": "Contact Info"
+                },
+                "path": "/contact-us-form/",
+                "plainTextSummary": "Visitors can use the form on this page to submit questions or feedback.&nbsp;\r\nStore is located in...",
+                "contactFields": [
+                  "fullname",
+                  "phone"
+                ]
+              }
+            },
+            {
+              "node": {
+                "__typename": "ExternalLinkPage",
+                "entityId": 8,
+                "name": "Youtube page",
+                "parentEntityId": null,
+                "seo": {
+                  "pageTitle": "",
+                  "metaKeywords": "",
+                  "metaDescription": ""
+                },
+                "link": "https://www.youtube.com/"
+              }
+            },
+            {
+              "node": {
+                "__typename": "RawHtmlPage",
+                "entityId": 9,
+                "name": "Random page",
+                "parentEntityId": null,
+                "seo": {
+                  "pageTitle": "",
+                  "metaKeywords": "",
+                  "metaDescription": ""
+                },
+                "path": "/random-page/",
+                "plainTextSummary": "This is a random page.\r\n\r\n\r\n\r\n\r\nThis is the body of a random page."
+              }
+            }
+          ]
+        }
+      }
+    }
+  }
+}
+```
+
+</Tab>
+</Tabs>
+
+### Rendered regions
+
+The following example retrieves rendered widgets on plain-text pages.
```

Comment:
```suggestion
The following example retrieves rendered widgets on plain-text pages. This will return the rendered HTML content of any widgets which have [Placements](https://developer.bigcommerce.com/api-docs/store-management/widgets/tutorials/widgets-tutorial#create-a-placement) in a given page's Regions.
```